www/install/sql/centreon/Update-DB-20.10.0-beta.2.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `platform_topology` ADD COLUMN `hostname` varchar(255) NULL AFTER `address`;
```

Comment:
kindly, if you can resolve the hostname and add them for remote or central in the upgrade script